spec/lib/terraforming/resource/iam_group_spec.rb

```diff
@@ -27,7 +27,15 @@ module Resource
       end
 
       before do
-        client.stub_responses(:list_groups, groups: groups)
+        client.stub_responses(:list_groups, [{
+                                               groups: [groups[0]],
```

Comment:
__[rubocop]__
* Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.